java/client/test/org/openqa/selenium/remote/tracing/opentelemetry/TracerTest.java

```diff
@@ -58,9 +58,12 @@
   public void shouldBeAbleToCreateATracer() {
     List<SpanData> allSpans = new ArrayList<>();
     Tracer tracer = createTracer(allSpans);
+    long timeStamp = 1593493828L;
 
     try (Span span = tracer.getCurrentContext().createSpan("parent")) {
       span.setAttribute("cheese", "gouda");
+      span.addEvent("Grating cheese");
```

Comment:
Break out tests for events into their own tests rather than placing them in other ones. That makes it easier for us to figure out where problems lie and to do a TDD-driven implementation over new APIs.

Reply:
Thank you Simon. I have addressed the changes requested in the next commit.